bindings/capacitorjs/tsconfig.json

```diff
@@ -0,0 +1,32 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+{
+  "compilerOptions": {
+    "allowUnreachableCode": true,
+    "declaration": true,
+    "esModuleInterop": true,
+    "lib": ["dom", "es2017"],
+    "module": "esnext",
+    "moduleResolution": "node",
+    "noFallthroughCasesInSwitch": true,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "outDir": "dist/esm",
+    "pretty": true,
+    "sourceMap": false,
+    "target": "es2017"
+  },
+  "files": [
+    "src/index.ts",
+    "src/definitions.ts",
+    // "src/lib/index.ts",
+    // "src/lib/bindings.ts",
+    // "src/lib/AccountManager.ts",
+    // "src/lib/Account.ts",
+    // "src/lib/MessageHandler.ts",
+    // "src/types/index.ts"
```

Comment:
Can we remove these lines if they are commented out?

Reply:
removed!